Sources/UXKit/AppKit/UXTableView-AppKit.swift

```diff
@@ -84,25 +116,35 @@
       reloadData(forRowIndexes: IndexSet.setForRowsInPathes(indexes),
                  columnIndexes: IndexSet(integer: 0))
     }
+    
+    func numberOfRows(inSection: Int) -> Int {
+        return self.numberOfRows
+    }
   }
 
-  public extension NSTableView {
+
+ extension UXTableView {
+    open override func edit(withFrame rect: NSRect, editor textObj: NSText, delegate: Any?, event: NSEvent) {
+        textObj.becomeFirstResponder()
+    }
+}
+  public extension UXTableView {
     
     /// UIKit compat method for `makeView(withIdentifier:owner:)`. This one
     /// passes `nil` as the owner.
     func dequeueReusableCell(withIdentifier identifier: String)
-         -> UXView?
+         -> UXTableViewCell?
```

Comment:
If we really want to do that (AppKit doesn't have this requirement), we should at least add an assertion if something different is returned. Like:
```
let  cellView = make....
assert(cellView is UXTableViewCell)
return cellView as? UXTableViewCell
```